manifests/params.pp

```diff
@@ -1,11 +1,8 @@
 #Class: fetchcrl::params
 class fetchcrl::params {
 
-  if $::osfamily == 'RedHat' and $::operatingsystemmajrelease == '5' {
-    $pkgname = fetch-crl3
-  } else {
-    $pkgname = fetch-crl
-  }
+  $pkgname = fetch-crl
```

Comment:
I'm surprised this isn't quoted and yet works.

Reply:
https://puppet.com/docs/puppet/5.5/lang_data_string.html#bare-words 
indeed should not work, changing.